packs/data/feats.db/maneuvering-spell.json

```diff
@@ -25,7 +25,7 @@
         },
         "rules": [],
         "source": {
-            "value": "Pathfinder #167: Ready? Fight!"
+            "value": "Fists of the Ruby Phoenix Hardcover, Pathfinder #167: Ready? Fight!"
```

Comment:
If there are no changes, the source is typically left alone.

Reply:
Thanks, fixed